rider-cdi/src/main/java/com/github/database/rider/cdi/DBUnitInterceptorImpl.java

```diff
@@ -25,28 +11,44 @@
 import com.github.database.rider.core.leak.LeakHunterFactory;
 import com.github.database.rider.core.util.AnnotationUtils;
 
+import javax.enterprise.inject.spi.CDI;
+import javax.inject.Inject;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+import javax.transaction.UserTransaction;
+import java.io.Serializable;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Created by pestano on 26/07/15.
  */
 @Interceptor
 @DBUnitInterceptor
 public class DBUnitInterceptorImpl implements Serializable {
 
+    private static final List<String> TEST_ANNOTATION_NAMES = Arrays.asList("Test", "Give", "When", "Then");
```

Comment:
Shouldn't it be `Given` (Cucumber, https://www.tutorialspoint.com/cucumber/cucumber_annotations.htm)?

Reply:
Thanks!